### ISSUE TITLE
Is it possible to reduce the input image size to increase efficient of the barcode scanner?

### ISSUE THREAD

**Issue by room101b** [EddyVerbruggen/nativescript-plugin-firebase#1442]
Hi, I have switched from the nativescript-barcode scanner to the ML scanner because I need to embed  the image and provide controls to end multiple scans on an Android phone. The ML scanner works but is far more inconsistent than the nativescript-barcode. It sometimes seems to get stuck, it won't focus, or focuses the camera incorrectly, and it needs more light.

The width and height parameters appear to reduce the size of the image on the screen but still using the full input from the camera. Is it possible to increase the reliability and efficiency of the camera, maybe by reducing the input image size?

**Comment by EddyVerbruggen** [comment]
Yes, please go ahead and play a little with the source code and see what works best for you. I'd be happy to learn what you've learned by doing that, so we may expose some configuration.

**Comment by room101b** [comment]
I have done some machine learning and neural networks before. Can you tell me a little about the structure of the neural network, the input size, how it was trained, and what it was trained on?
